### PR TITLE
zarwin.is-a.dev

### PR DESCRIPTION
**Pull request by zarwin**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: https://zarwin.github.io/zarwin.is-a.dev
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/zarwin.json
Comment (VaibhavSys) [inline domains/zarwin.json:7]: A `,` is missing.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/zarwin.json
Comment (zarwin) [reply domains/zarwin.json]: Where ?? I coudn't get that

**Review by zarwin — COMMENTED** [review]
Done the Suggested changes Please re-review my PR

**Review by VaibhavSys — APPROVED** [review]
Now wait for a maintainer to review and merge.

**Comment by academicgit** [comment]
Please Merge this PR

**Comment by mtgsquad** [comment]
Hi, i'm incredibly sorry for the long wait times. I apologise for the wait.

**Review by mtgsquad — APPROVED** [review]

**Comment by zarwin** [comment]
> Hi, i'm incredibly sorry for the long wait times. I apologise for the wait.

No Problem and keep up the good work!

**Pull request merged**
